apps/crn-server/src/routes/research-outputs.route.ts

```diff
@@ -11,27 +12,59 @@ import {
 import Boom from '@hapi/boom';
 import { Response, Router } from 'express';
 import { ResearchOutputController } from '../controllers/research-outputs';
+import { TeamDataProvider } from '../data-providers/teams.data-provider';
+import { WorkingGroupDataProvider } from '../data-providers/working-groups.data-provider';
 import {
   validateResearchOutputParameters,
   validateResearchOutputPostRequestParameters,
   validateResearchOutputPostRequestParametersIdentifiers,
   validateResearchOutputRequestQueryParameters,
   validateResearchOutputPutRequestParameters,
+  validateResearchOutputDraftFetchOptions,
 } from '../validation/research-output.validation';
 
 export const researchOutputRouteFactory = (
   researchOutputController: ResearchOutputController,
+  workingGroupDataProvider: WorkingGroupDataProvider,
+  teamDataProvider: TeamDataProvider,
 ): Router => {
   const researchOutputRoutes = Router();
 
   researchOutputRoutes.get(
     '/research-outputs',
     async (req, res: Response<ListResearchOutputResponse>) => {
       const { query } = req;
-
-      const options = validateFetchOptions(query);
-
-      const result = await researchOutputController.fetch(options);
+      let result;
+
+      if (query.status) {
```

Comment:
so if no status is sent we will just return zero results? not sure if i am reading it right since there are no tests